src/components/cc-invoice-table/cc-invoice-table.js

```diff
@@ -70,13 +68,29 @@ export class CcInvoiceTable extends LitElement {
         };
       })
       .sort(sortBy('emissionDate', true));
+  }
+
+  render () {
+
+    // NOTE: This value is arbitrary, we don't have a better solution for now
+    // It's a bit more than the width of the table in french (which is the largest) and with both links (download and pay)
+    // The table width is mostly stable since the with of the amount is fixed and the rest is almost always the same number of characters
```

Comment:
fix: I don't understand this sentence: 'since the with of the amount'?

Reply:
This note already existed before, I simply moved it but I did not read it at the time.

I think there's a typo and the sentence should be 
```suggestion
    // The table width is mostly stable since the width of the amount is fixed and the rest is almost always the same number of characters
```

Meaning the width of the figure displayed is always the same I guess :thinking: 
I'll fix the typo, thanks for this ;)